common/config/api.go

```diff
@@ -38,7 +38,7 @@ func APIFlags() []cli.Flag {
 		},
 		cli.StringFlag{
 			Name:   FLAG_AWS_REGION,
-			Value:  "us-west-2",
+			Value:  DefaultAWSRegion,
```

Comment:
Should this be all caps or no since it's not an ENV?

Reply:
We've being making all of our `const` variables in all caps because that's what they do in python. Turns out, that's not what they do in Go; they just use upper camel case. There's something to be said for fixing all of these at once though...